Moto/G8Plus/res/values/config.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_dozeAfterScreenOff">false</bool>
+    <bool name="config_dynamic_bind_ims">true</bool>
+    <string name="config_ims_package">org.codeaurora.ims</string>
+    <bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
+    <bool name="config_setColorTransformAccelerated">true</bool>
+    <bool name="config_pinnerCameraApp">true</bool>
+    <bool name="config_useDevInputEventForAudioJack">true</bool>
+    <integer name="config_bluetooth_operating_voltage_mv">3300</integer>
+    <integer name="config_screenBrightnessDoze">17</integer>
```

Comment:
Sounds very high. You probably want `1` here.
In all likelihood, `17` is for 12bits brightness, but aosp only has 8 bits.